github/commitstatus.go

```diff
@@ -127,6 +135,7 @@ func (s CommitStatus) Add(sha, state, targetURL, description string) error {
 			break
 		}
 		s.log.Info("Sleeping for", "duration", timeToSleep, "reason", reason)
+		s.sleepFn(timeToSleep)
```

Comment:
See here: https://github.com/Pix4D/cogito/pull/133/files#r1286852612

I think this is still problematic.

Reply:
Thanks. Fixed.